runtime/bindings/python/tests/test_ngraph/test_reduction.py

```diff
@@ -3,7 +3,7 @@
 
 import numpy as np
 import pytest
-from _pyngraph import PartialShape, Dimension
+# from _pyngraph import PartialShape, Dimension
```

Comment:
Good catch it should be `from ov import PartialShape, Dimension` if to be used.

Reply:
may be *from openvino import PartialShape, Dimension*?